lib/dialects/sqlite/query.js

```diff
@@ -295,6 +302,50 @@ class Query extends AbstractQuery {
     });
   }
 
+  parseConstraintsFromSql(sql) {
+    let constraints = sql.split('CONSTRAINT ');
+    let referenceTableName, referenceTableKeys, update_action, delete_action;
+    constraints.splice(0, 1);
+    constraints = constraints.map(constraintSql => {
+      //Parse foreign key snippets
+      if (constraintSql.indexOf('REFERENCES') !== -1) {
+        //Parse out the constraint condition form sql string
+        update_action = constraintSql.match(/ON UPDATE (CASCADE|SET NULL|RESTRICT|NO ACTION|SET DEFAULT){1}/);
+        delete_action = constraintSql.match(/ON DELETE (CASCADE|SET NULL|RESTRICT|NO ACTION|SET DEFAULT){1}/);
```

Comment:
Apologies. Looks like MSSQL is the culprit here. I'm using MSSQL stored procedure [sp_helpconstraint](https://msdn.microsoft.com/en-us/library/ms176038.aspx) and hence can't format the results. I used the same snake_case for other dialects aswell.
Since default constraint in case of mssql are not part of ISO standard, i can't query on `INFORMATION_SCHEMA.TABLE_CONSTRAINTS` as we query in case of mysql and postgresql.
We could convert the snake_case to camelCase in mssql query.js and use sql aliases in case of mysql postgres?